WordPress/Classes/ViewRelated/Blog/Site Monitoring/SiteMonitoringView.swift

```diff
@@ -2,25 +2,105 @@ import SwiftUI
 import UIKit
 
 struct SiteMonitoringView: View {
+    @StateObject var viewModel: SiteMonitoringViewModel
+
     var body: some View {
-        Text("Site Monitoring")
+        main.navigationTitle(viewModel.selectedTab.title)
+            .navigationBarTitleDisplayMode(.inline)
+            .apply(addToolbarTitleMenu)
+    }
+
+    @ViewBuilder
+    private var main: some View {
+        switch viewModel.selectedTab {
+        case .metrics:
+            WebView(url: viewModel.metricsURL!)
+        case .phpLogs:
+            phpLogs
+        case .webServerLogs:
+            webServerLogs
+        }
+    }
+
+    @ViewBuilder
+    private var phpLogs: some View {
+        List {
+            Text("PHP Logs")
+        }
+    }
+
+    @ViewBuilder
+    private var webServerLogs: some View {
+        List {
+            Text("Web Server Logs")
+        }
+    }
+
+    @ViewBuilder
+    private func addToolbarTitleMenu<T: View>(_ view: T) -> some View {
+        if #available(iOS 16, *) {
```

Comment:
I would suggest making this whole feature only available on iOS 16. The number of users on iOS 15 is negligible.